architecture/core_objects/builds.adoc

```diff
@@ -171,55 +171,12 @@ You can define multiple Docker registry entries in this file. Alternatively, you
 can also add authentication entries to this file by running the `docker login`
 command. The file will be created if it does not exist.
 
-Kubernetes provides the
-https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/secrets.md[Secret]
-resource, which is used to store your configuration and passwords. You must
-first create the `*Secret*` before builds can use your *_.dockercfg_* file for
-pushing the output image:
-
-====
-----
-$ osc create -f secret.json
-----
-====
-
-The `*data*` field for the `*Secret*` object must contain the `*dockercfg*` key
-with the value set to the base64-encoded content of the *_.dockercfg_* file. For
```

Comment:
I seem to recall it being important that the key be named specifically `dockercfg` (no dot prefix) in the `data` stanza. Is this no longer the case? The `bundle-secret` command seems to use whatever the file name given is, in the below example's case `.dockercfg`:

```
    "data": {
        ".dockercfg": "ewoJImh0dHBzOi8vaW5kZXguZG9ja2VyLmlvL3YxLyI6IHsKCQkiYXV0aCI6ICJZV1JsYkd4aGNHVTZSMmxoYm5SaWIyMWlJVEU9IiwKCQkiZW1haWwiOiAiYWRlbGxhcGVAcmVkaGF0LmNvbSIKCX0KfQ=="
```